collection-scripts/version

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+function version() {
+  # get version from image
+  version=$( \
+    oc status | grep '^pod' | \
+    sed -n -r -e 's/.*([[:digit:]]+\.[[:digit:]]+(:?\.[[:digit:]])?(:?-[^@]+)?).*/\1/p' \
```

Comment:
when I run this on my notebook (after I logged in the cluster) I get empty response. What is it supposed to print out?

Reply:
I updated this script and the expected string from this script is the following:
~~~
opendatahub/must-gather
1.15.0-10
~~~